YIF.Core.Service/Concrete/Services/SuperAdminService.cs

```diff
@@ -214,17 +214,22 @@ public async Task<ResponseApiModel<AuthenticateResponseApiModel>> AddSchoolAdmin
 
             return result.Set(new AuthenticateResponseApiModel(token, refreshToken), true);
         }
-
+        
         public async Task<ResponseApiModel<DescriptionResponseApiModel>> DeleteInstitutionOfEducationAdmin(string adminId)
         {
             var result = new ResponseApiModel<DescriptionResponseApiModel>();
-            var ch = await _institutionOfEducationAdminRepository.GetUserByAdminId(adminId);
-            if (ch == null)
+            var admin = await _institutionOfEducationAdminRepository.GetUserByAdminId(adminId);
+            if (admin == null)
             {
-                throw new NotFoundException($"{_resourceManager.GetString("UserWithSuchIdNotFound")}: {adminId}");
+                throw new NotFoundException(_resourceManager.GetString("IoEAdminNotFound"));
             }
-            await _userRepository.Delete(ch.User.Id);
-            return result.Set(new DescriptionResponseApiModel("User IsDeleted was updated"), true);
+
+            var searchUser = _userManager.FindByIdAsync(admin.UserId);
```

Comment:
Add await, because:
https://stackoverflow.com/questions/32239661/await-vs-task-result-in-an-async-method